File: HollypocketBackend/Models/Account.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace HollypocketBackend.Models
{
    public class Account
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        [Required]
        public string Email { get; set; }
        public List<Address> Addresses { get; set; }
        public string PhoneNumber { get; set; }
        public string Name { get; set; }
        public string Gender { get; set; }
        [Required]
        public string Password { get; set; }
        public string Token { get; set; }
        public AccountType AccountType { get; set; }
        public bool IsDeleted { get; set; }
    }
    public class Address
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Value { get; set; }
        public string PhoneNumber { get; set; }
    }
}

public enum AccountType
{
    Admin = 0,
    Normal = 1,
    Employee = 2
}

File: HollypocketBackend/mongodb/account.js
db.createCollection("Accounts");

db.Accounts.insertMany([
  {
    Email: "admin@gmail.com",
    Addresses: [
      {
        Name: "dinh mai",
        Value: "123231",
        PhoneNumber: "212",
      },
    ],
    PhoneNumber: "0987654742",
    Name: "Admin",
    Gender: "male",
    Password: "password",
    Token: null,
    AccountType: 0,
    IsDeleted: false,
  },
  {
    Email: "user@gmail.com",
    Addresses: [
      {
        Name: "dinh mai",
        Value: "123231",
        PhoneNumber: "212",
      },
    ],
    PhoneNumber: "0987654742",
    Name: "user",
    Gender: "male",
    Password: "password",
    Token: null,
    AccountType: 1,
    IsDeleted: false,
  },
  {
    Email: "employ@gmail.com",
    Addresses: [
      {
        Name: "dinh mai",
        Value: "123231",
        PhoneNumber: "212",
      },
    ],
    PhoneNumber: "0987654742",
    Name: "employee",
    Gender: "male",
    Password: "password",
    Token: null,
    AccountType: 2,
    IsDeleted: false,
  },
]);
